lib/airrecord/table.rb

```diff
@@ -186,17 +186,18 @@ def destroy
 
     def serializable_fields(fields = self.fields)
       Hash[fields.map { |(key, value)|
-        if association(key)
-          assocs = value.map { |assoc|
-            assoc.respond_to?(:id) ? assoc.id : assoc
-          }
-          [key, assocs]
-        else
-          [key, value]
-        end
-      }]
-    end
-
+             if association(key)
+               value = [value] unless value.respond_to?(:map)
```

Comment:
`Array.new(value)` is my preferred way of enforcing an array, since it will do the right thing for an existing array. Checking for `map` is not a reliable way to detect an array.